src/pages/study/courses/LectureDetailContent.tsx

```diff
@@ -0,0 +1,183 @@
+import { ExternalLink } from 'lucide-react'
+
+import { PlatformBadge } from '@/components/common/badge'
+import type { Lecture } from '@/pages/study/courses'
+import { formatDateTime } from '@/utils'
+import { twClassName } from '@/utils/getTwClassName'
+
+interface LectureDetailContentProps {
+  lecture: Lecture
+}
+
+type DifficultyProps = {
+  desc: string
+  style: string
+}
+
+const DIFFICULTY: Record<string, DifficultyProps> = {
+  HARD: { desc: '어려움', style: 'bg-red-100 text-red-700' },
+  NORMAL: { desc: '보통', style: 'bg-yellow-100 text-yellow-700' },
+  EASY: { desc: '쉬움', style: 'bg-green-100 text-green-700' },
+}
+
+const getDifficulty = ({
+  difficulty,
+  className,
+}: {
+  difficulty: string
+  className?: string
+}) => {
+  const baseClass = 'inline-block rounded px-2 py-1 text-lg'
+
+  const difficultyData = DIFFICULTY[difficulty] || {
+    desc: '-',
+    style: 'bg-gray-100 text-gray-700',
+  }
+  const difficultyStyle = difficultyData.style
+  const difficultyDesc = difficultyData.desc
+
+  return (
+    <span className={twClassName(baseClass, difficultyStyle, className)}>
+      {difficultyDesc}
+    </span>
+  )
+}
```

Comment:
함수로 관리하는 방식보다는 컴포넌트화해서 관리해도 괜찮을 것 같아요!
```suggestion
const DIFFICULTY_CONFIG: Record<string, { desc: string; style: string }> = {
  HARD: { desc: '어려움', style: 'bg-red-100 text-red-700' },
  NORMAL: { desc: '보통', style: 'bg-yellow-100 text-yellow-700' },
  EASY: { desc: '쉬움', style: 'bg-green-100 text-green-700' },
}

const DifficultyBadge = ({ difficulty, className }: { difficulty?: string, className?: string }) => {
  const baseClass = 'inline-block rounded px-2 py-1 text-sm' 
  const { desc, style } = DIFFICULTY_CONFIG[difficulty || ''] || {
    desc: '-',
    style: 'bg-gray-100 text-gray-700',
  }

  return (
    <span className={twClassName(baseClass, style, className)}>
      {desc}
    </span>
  )
}
```